SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -109,7 +109,9 @@ Worker_RequestId USpatialSender::CreateEntity(USpatialActorChannel* Channel)
 	const FClassInfo& Info = ClassInfoManager->GetOrCreateClassInfoByClass(Class);
 
 	WorkerRequirementSet AuthoritativeWorkerRequirementSet = SpatialConstants::UnrealServerPermission;
-	if (!Class->WorkerAssociation.IsEmpty())
+
+	const USpatialGDKSettings* SpatialGDKSettings = GetDefault<USpatialGDKSettings>();
+	if (SpatialGDKSettings->bUsingOffloading && !Class->WorkerAssociation.IsEmpty())
```

Comment:
Just a thought, can the inside of the conditional just be one line instead?

```
AuthoritativeWorkerRequirementSet = { { Class->WorkerAssociation } };
```

Reply:
Just a thought, can the inside of the conditional just be one line instead?

```
AuthoritativeWorkerRequirementSet = { { Class->WorkerAssociation } };
```

Reply:
This worked am making an update now